.eslintrc.json

```diff
@@ -36,6 +36,38 @@
         "args": "after-used",
         "argsIgnorePattern": "^_"
       }
-    ]
+    ],
+    "import/order": [
+      "error",
+      {
+        "pathGroups": [
+          {
+            "pattern": "{react,jotai,jotai/*}",
+            "group": "external",
+            "position": "before"
+          },
+          {
+            "pattern": "modules/**",
+            "group": "builtin",
+            "position": "after"
+          },
+          {
+            "pattern": "{api,abis,common,constants,legacy,lib,pages,types,utils}/**",
+            "group": "internal",
+            "position": "before"
+          }
```

Comment:
Should we create a group for `@cowprotocol`, i would put it, in between externals, and `legacy`

Reply:
shouldn't legacy be its own group? this way we see together all the importants that should dissapear

Reply:
Added separated group. Check it out in the PR description